build.zig

```diff
@@ -185,6 +190,18 @@ pub fn build(b: *Builder) !void {
         run_prover.addArgs(&[_][]const u8{ "-d", b.fmt("{s}/bin", .{b.install_path}) });
     }
 
+    // Add the tools cli executable
+    const tools_cli_exe = b.addExecutable(.{
+        .name = "zeam-tools",
+        .root_source_file = b.path("pkgs/tools-cli/src/main.zig"),
+        .optimize = optimize,
+        .target = target,
+    });
+    tools_cli_exe.root_module.addImport("enr", enr);
+    tools_cli_exe.root_module.addImport("build_options", build_options.createModule());
+    tools_cli_exe.root_module.addImport("simargs", zigcli.module("simargs"));
+    b.installArtifact(tools_cli_exe);
+
```

Comment:
the tool should have its own build, as we only want to build it if needed. You can add a `all` target that would build all the tools in the `pkgs/tools-cli` directory